qadence/backends/pyqtorch/backend.py

```diff
@@ -30,6 +30,9 @@
 from .config import Configuration
 from .convert_ops import convert_block, convert_observable
 
+DEVICE = torch.device("cuda") if torch.cuda.is_available() else torch.device("cpu")
+torch.set_default_device(DEVICE)
```

Comment:
I thought this would move all the Torch objects to the device without user intervention.

Reply:
I'm not sure this is a good idea.
Not to mention if you're on the cluster you might get unexpected results since there will be multiple gpus, so this could make multi-gpu training difficult.

this looks to me to be something the user might write, but not us.

Reply:
agreed and removed